comm/src/test/scala/coop/rchain/p2p/ConnectToBootstrapSpec.scala

```diff
@@ -61,45 +52,28 @@ class ConnectToBootstrapSpec
 
     it("should log on ERROR and return error when failed connecting") {
       // given
-      transportLayerEff.setResponses(kp(failEverything))
+      // transportLayerEff.setResponses(kp(failEverything))
```

Comment:
Why do you want to keep this?

Reply:
forgot about those tests, they are turned on once again, thx